pom.xml

```diff
@@ -394,6 +405,11 @@
                 <artifactId>jakarta.xml.bind-api</artifactId>
                 <version>2.3.3</version>
             </dependency>
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>1.3.1</version>
+            </dependency>
             <dependency>
```

Comment:
Need `javax.annotation-api` as it was removed in java 11
https://blogs.perficient.com/2019/05/15/the-java-commons-annotations-was-removed-in-jdk-11/

Reply:
The blog post is not correct.  It would be `jakarta.annotation-api` that should be used as the replacement.  It is already included in the dependency management.